extensions/amp-bind/0.1/bind-expr.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright 2016 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {parser} from './bind-expr-impl';
+
+export function evaluateBindExpr(expr, data) {
```

Comment:
Jison doesn't support it natively, but I could build a custom one in the grammar. Not sure how much of a speed-up we'd get but I'll try it out.